src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  return (firstValue) => {
+    return (secondValue) => {
+      calls++;
+
+      if (calls <= 3 || calls % 2 === 1) {
```

Comment:
Not sure if it is necessary here, but you can simplify the condition
```suggestion
      if (calls <= 3 || calls % 2) {
```